src/reposetories/userRepository.ts

```diff
@@ -7,3 +7,25 @@ export const createUser = async (userData: any) => {
 export const getUserByEmail = async (email: string) => {
     return await User.findOne({ where: { email } }); // Retrieve the user by email
 };
+
```

Comment:
it's better to have these in separate PR but it's okay don't change it just for the future it's a better practice to have small PRs that have one gaol.